### ISSUE TITLE
Coordinate translation isn't working properly

### ISSUE THREAD

**Issue by autodidaddict** [wascc/wasmdome-web#6]
The domain model (event sourcing) treats the origin of the arena of (0,0) to be the bottom left corner of the arena, or the most _Southwesterly_ point on the game board. This means that a position of (0,H) is the most northwest, and (H, W) is the most northeast where H and W are the board height and width.

When the mechs are displayed on the game board inside a CSS grid, the grid requires columns and rows, where col 0 and row 0 are the top left. 

In its current state, the translation is not working. I also wasn't able to figure out how to turn on internal grid lines to make troubleshooting this easier.

The module`Core.Mech` has a `to_gridpiece` function that is not currently doing translation properly, and it's causing a cascading failure in the tests (which are commented out right now)

tl;dr:

`Core.Mech.to_gridpiece` needs to convert the `{x,y}` tuple into a `column` and `row` property on the grid piece.
